hydpy/models/rconc_nash.py

```diff
@@ -0,0 +1,36 @@
+# -*- coding: utf-8 -*-
+"""Calculate runoff concentration by linear storage cascade model ('Nash cascade').
+
+|rconc_nash| is a submodel that supplies its main model with the
+calculation of the runoff concentration by the storage cascade approach.
+
+The integration test of the application model |hland_v3| uses the
+|rconc_nash| submodel.
+"""
+# import...
+# ...from HydPy
+from hydpy.exe.modelimports import *
+from hydpy.interfaces import rconcinterfaces
+from hydpy.models.rconc import rconc_model
+
+
+class Model(rconc_model.Sub_RConcModel, rconcinterfaces.RConcModel_V1):
+    """Model that calculates runoff concentration by storage cascade approach"""
```

Comment:
"."

Also, I am unsure about this sentence's grammar.

Reply:
Grammar checker says it's correct but proposed a better style: A model that calculates runoff concentration using the storage cascade approach.